git_machete/cmd.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Hmmm that's a potentially risky move, there are two alternatives:
* running `python` and then raising an error if it happens to be Python 2, see what we do cmd.py
* running `python3` and then ending up with `file not found` if the system doesn't happen to have `python3` executable, only `python` (which might correspond to Python 3 btw :trollface:)... I've seen that on a Windows recently ;p

Reply:
Was there any specific reason for this change? 🤔

Reply:
hmm, that breaks rpm package build:
<img width="1124" alt="Screenshot 2021-08-25 at 09 13 54" src="https://user-images.githubusercontent.com/73847352/130743737-f6ddcbbc-1730-4f96-a631-f650df30bf88.png">

Reply:
according to : https://fedoraproject.org/wiki/Changes/Make_ambiguous_python_shebangs_error
this was previously a warning only

Reply:
Okay! didn't know about that 😯